gspread/utils.py

```diff
@@ -581,7 +581,7 @@ def fill_gaps(
         return [[]]
 
 
-def cell_list_to_rect(cell_list: List["Cell"]) -> List[List[Optional[str]]]:
+def cell_list_to_rect(cell_list: List[Any]) -> List[List[Optional[str]]]:
```

Comment:
Usage of literal "Cell" type is recognized as error by linter. But importing of Cell class  produce circular imports. I think we can refactor utils.py in later and place utilities that depends on specific classes in same modules. Now I replace it Cell with Any for CI passing

Reply:
Hi yes I noticed similar issues when doing Worksheet.py, to solve this you can use the following trick:

- at the top of the file check for the value of `TYPE_CHECKING` 
- if true, then import what you need regardless of circular dependencies
- keep your code as it was by putting double quotes around the class name

The check at the top of the file is only valid when mypy is running, that ensures the code runs and allows you to check the types.

You can check on my branch `feature/add_type_hints_worksheets` i do that for the http client